source/api/auth/github.ts

```diff
@@ -74,7 +73,7 @@ export const generateAuthToken = async (req: Request, res: Response, ___: NextFu
   res.cookie("jwt", authToken, { domain: ".peril.systems" })
 
   if (redirect) {
-    res.redirect(redirect)
+    res.redirect(redirect + "?perilJWT=" + encodeURIComponent(authToken))
```

Comment:
mainly because I need this for dev, as it gets stored in a cookie anyway, I'm ok with this.